docs/body.html

```diff
@@ -230,6 +221,24 @@ <h3>Resources</h3>
               >.
             </p>
           </div>
+
+          <div class="callout__list">
+            <i class="fab fa-solid fa-calendar-days"aria-hidden="true"></i>
```

Comment:
I think this the equivalent v5 icon:
```suggestion
            <i class="fa fa-solid fa-calendar-alt" aria-hidden="true"></i>
```

Reply:
ah thanks. I just saw the other icons were added as svgs and was working on adding on adding this one as well. Is that preferred?

Reply:
I think those must be old; I don't see them referenced anywhere.